.github/workflows/coverage.yml

```diff
@@ -23,13 +20,29 @@ jobs:
     - uses: ./.github/actions/testing-setup
 
     - name: Compile unit testing
-      run: make -j build/unit/MOM6
+      run: make -j build/unit/MOM_unit_tests
 
     - name: Run unit tests
-      run: make unit.cov.upload
+      run: make run.cov.unit
+
+    - name: Report unit test coverage to CI
```

Comment:
Minor issue but it will help avoid confusion: could we name these two steps differently, e.g. "Report unit test coverage for PR" and "Report unit test covereage for commit"? Otherwise, the CI shows the same thing with a no-entry sign.

![image](https://user-images.githubusercontent.com/5859571/181774081-3ae5cf00-534d-483a-bef7-47e6a6de87fb.png)

Reply:
I agree, I half-considered it at the time.  I'll put in the change.

Reply:
I see the patch for this